users_ldap_populate/view/users_ldap.xml

```diff
@@ -6,7 +6,7 @@
             <field name="model">res.company</field>
             <field name="inherit_id" ref="auth_ldap.company_form_view"/>
             <field name="arch" type="xml">
-                <xpath expr="//form[@string='LDAP Configuration']" position="inside">
+                <xpath name="populate_ldap" expr="//field[@name='ldaps']/form" position="inside">
```

Comment:
My bad, `name` is perfectly OK.
I was just checking if you guys were paying attention :wink: